samza-core/src/test/scala/org/apache/samza/storage/TestTaskStorageManager.scala

```diff
@@ -84,7 +84,7 @@ class TestTaskStorageManager extends MockitoSugar {
     val ssp = new SystemStreamPartition(ss, partition)
     val storeDirectory = StorageManagerUtil.getStorePartitionDir(TaskStorageManagerBuilder.defaultLoggedStoreBaseDir, loggedStore, taskName, TaskMode.Active)
     val storeFile = new File(storeDirectory, "store.sst")
-    val offsetFile = new File(storeDirectory, StorageManagerUtil.OFFSET_FILE_NAME)
```

Comment:
1) Do you have a test for only having the old offset file with the old format?
2) Do you have a test for writing the old offset file with the old format?

Reply:
yes, sending for review in a separate pr